server.html

```diff
@@ -85,92 +92,10 @@
           and email credentials needed to send outgoing email.</p>
         </section>
         <section>
-          <a name="troubleshooting"><h3>Troubleshooting instructions</h3></a>
-          <p>The instructions here are an abbreviated version of the
-          instructions in
-          the <a href="https://github.com/zulip/zulip/blob/master/README.prod.md">Zulip
-          in production README</a>; you will want to consult those
-          instructions once you finish the steps here or if you run
-          into any trouble.
-          <p>
-        </section>
-        <section>
           <a name="install"><h3>Installation</h3></a>
-	  <section>
-	    <p>The installation process should be executed as the root user. Run:
-<div class="terminal">
-  sudo -i
-            </div> 
-	  </section>
-          <section>
-            <a name="download"><h4>Download the tarball</h4></a>
-            <p>Download
-            the <a href="https://www.zulip.com/dist/releases/zulip-server-1.3.12.tar.gz">binary
-            tarball</a>, Verify its sha1sum matches, extract it, and
-            move the resulting directory to /root/zulip:</p>
-            <div class="terminal">
-              cd /root<br />
-              wget https://www.zulip.com/dist/releases/zulip-server-1.3.12.tar.gz<br />
-              echo "95f65a74f7447b5b4c46e7ed487fca50ff7a2c72 zulip-server-1.3.12.tar.gz" | sha1sum -c<br />
-              tar -xf zulip-server-1.3.12.tar.gz<br />
-              mv zulip-server-1.3.12 zulip<br />
```

Comment:
Understood. I kept the sha1sums and got rid of the rest of the duplicated content.